pkg/apis/nmstate.io/v1/types.go

```diff
@@ -174,14 +174,27 @@ type BridgePort struct {
 	// TODO description
 	// TODO: Mandatory?
 	// TODO: possible values
-	Type string `json:"type"`
+	Type string `json:"type,omitempty"`
 	// TODO description
 	// TODO: Mandatory?
 	// TODO: possible values?
-	VlanMode string `json:"vlanMode"`
+	VlanMode string `json:"vlanMode,omitempty"`
```

Comment:
These are not used with Linux bridge, right? Only OVS. Could you please check nmstate schema in github.com/nmstate/nmstate to make sure we don't have unused attributes here? Or better, open an issue to track it.

Reply:
It's only of OVS https://github.com/nmstate/nmstate/blob/master/libnmstate/schemas/operational-state.yaml#L267, let's remove it.

but BridgePort is used for linux and OVS so we included it here.

Reply:
Mmm, let's go with this for now, but we should have Port and OVSPort later for the sake of documentation (unless we have feature parity)